packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/moduleinterops.js

```diff
@@ -0,0 +1,5 @@
+
+const a = { nameCollision:1 }
+module.exports.notDefault = 2
+module.exports.nameCollision = 3
+module.exports.default = a
```

Comment:
`yarn lint-fix` would add semicolons and space after colon if it were playing along.

Reply:
I got eslint running on fixtures too, but it had some destructive opinions. Also, didn't fix much. Prettier will do for now.

what it took to run eslint:
```
    "lint-fixtures": "find ./test/fixtures-*/**/*.js | xargs eslint --no-ignore",
    "lint-fixtures": "find ./test/fixtures-*/** -name '*.js' -exec eslint --no-ignore {} \\;",
```